app/src/main/java/com/github/multimatum_team/multimatum/DeadlineDetailsActivity.kt

```diff
@@ -0,0 +1,67 @@
+package com.github.multimatum_team.multimatum
+
+import android.content.Context
+import android.content.Intent
+import android.os.Bundle
+import android.widget.TextView
+import androidx.appcompat.app.AppCompatActivity
+import com.github.multimatum_team.multimatum.model.Deadline
+import com.github.multimatum_team.multimatum.model.DeadlineState
+import java.time.LocalDate
+import java.time.LocalDate.*
+import java.time.Period
+
+/**
+ * Classes used when you select a deadline in the list, displaying its details.
+ * In the future, It should have a delete and modify button to change the deadline.
+ */
+class DeadlineDetailsActivity : AppCompatActivity() {
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_deadline_details)
+
+        // Recuperate the Deadline from the intent
+        val title = intent.getStringExtra(EXTRA_TITLE)
+        val date = intent.getSerializableExtra(EXTRA_DATE) as LocalDate
+        val state = intent.getSerializableExtra(EXTRA_STATE) as DeadlineState
+
+        // Set the texts for the title and the date of the deadline
+        findViewById<TextView>(R.id.titleDeadlineDetails).text = title
+        findViewById<TextView>(R.id.dateDeadlineDetails).text = getString(R.string.DueTheX, date)
+
+        // Set the detail text to inform the user if it is due, done or the remaining time
+        val detailView = findViewById<TextView>(R.id.detailsDeadlineDetails)
+        when {
+            state == DeadlineState.DONE -> {
+                detailView.text = getString(R.string.done)
+            }
+            date.isBefore(now()) -> {
+                detailView.text = getString(R.string.isAlreadyDue)
+            }
+            else -> {
+                detailView.text =
+                    getString(R.string.DueInXDays, Period.between(now(), date).days.toString())
```

Comment:
Same comment as above. But since you are using it at several places you can simply create a value.